### PR TITLE
HARP-3815: Host examples on Github Pages.

### PR DESCRIPTION
**Pull request by amsgames94**

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE HTML>
+<html lang="en-US">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="refresh" content="0; url=https://heremaps.github.io/harp.gl/doc">
```
Comment (haraldF) [inline docs/index.html]: Does this have to be an absolute URL?

File: .travis.yml [hunk 2]
```diff
@@ -38,18 +38,17 @@ jobs:
 
     - name: "Build documentation"
       script: |
-        yarn run typedoc && \
-        echo -e 'include:\n  - "_*"' > doc/_config.yml
+        yarn run typedoc && cp ./docs/index.html ./dist && \
```
Comment (haraldF) [inline .travis.yml]: would you mind moving that to a new line? Two shell commands in the same line doesn't look good :)

Comment (haraldF) [inline .travis.yml]:
```suggestion
        yarn run typedoc && \
        cp ./docs/index.html ./dist && \
```

File: scripts/doc-snippets.ts [hunk 3]
```diff
@@ -47,7 +47,7 @@ function reindented(spaces: number, lines: string[]): string[] {
 const mkpath = require("mkpath");
 
 const sdkDir = path.resolve(__dirname, "..");
-const outDir = path.resolve(sdkDir, "dist/doc");
+const outDir = path.resolve(sdkDir, "dist/snippets");
```
Comment (haraldF) [inline scripts/doc-snippets.ts]: snippets is quite generic - `doc-snippets`?

File: docs/index.html (hunk 1 above)
Comment (amsgames94) [reply docs/index.html]: I guess not, but is there any good reason to change it to a relative URL?

File: .travis.yml (hunk 2 above)
Comment (amsgames94) [reply .travis.yml]: Already addressed in latest revision.

Comment (amsgames94) [reply .travis.yml]: Already addressed in latest revision.

File: scripts/doc-snippets.ts (hunk 3 above)
Comment (amsgames94) [reply scripts/doc-snippets.ts]: Will do!

**Pull request merged**